lib/models/os.coffee

```diff
@@ -304,6 +304,42 @@ getOsModel = (deps, opts) ->
 				throw new Error('No such version for the device type')
 			.asCallback(callback)
 
+	###*
+	# @summary Get an applications config.json
+	# @name getConfig
+	# @public
+	# @function
+	# @memberof resin.models.os
+	#
+	# @param {Number} Id - application id (number)
+	# @fulfil {Object} - application configuration as a JSON object
+	# @returns {Promise}
+	#
+	# @example
+	# resin.models.os.getConfig('MyApp').then(function(config) {
+	# 	fs.writeFile('foo/bar/config.json', JSON.stringify(config));
+	# });
+	#
+	# resin.models.os.getConfig(123).then(function(config) {
+	# 	fs.writeFile('foo/bar/config.json', JSON.stringify(config));
+	# });
+	#
+	#
+	# resin.models.os.download('MyApp', function(error, config) {
+	# 	if (error) throw error;
+	# 	fs.writeFile('foo/bar/config.json', JSON.stringify(config));
+	# });
+	###
+	exports.getConfig = (id, callback) ->
+		callback = findCallback(arguments)
```

Comment:
You don't need to do this unless you have optional arguments. With only required arguments, you know exactly which argument is the callback already, so you can just use it directly here.